pkg/test/v1test1/doc.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// +k8s:openapi-gen=true
+// +k8s:deepcopy-gen=package,register
+// +k8s:defaulter-gen=TypeMeta
+// +groupName=karpenter.test.sh
+package v1test1 // doc.go is discovered by codegen
+import (
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/kubernetes/scheme"
+)
+
+const (
+	Group = "karpenter.test.sh"
+)
+
+func init() {
+	gv := schema.GroupVersion{Group: Group, Version: "v1test1"}
+	v1.AddToGroupVersion(scheme.Scheme, gv)
```

Comment:
I don't think that you need this line anymore

Reply:
I think we still need it. I see errors if we don't have this.
`v1.GetOptions is not suitable for converting to "karpenter.test.sh/v1alpha1" in scheme "pkg/runtime/scheme.go:100"`